### PR TITLE
Chore: Trim snyk data to only include what webhint needs

### LINKED ISSUE #2100: Trim down snyk-snapshot.json

### PR DESCRIPTION
**Pull request by antross**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- ~Added/Updated related documentation.~
- ~Added/Updated related tests.~

## Short description of the change(s)

Fix #2100

Reduce size of bundled `snyk-snapshot.json` from 310KB to 12KB.

### CODE REVIEW

File: packages/hint-no-vulnerable-javascript-libraries/src/types.ts [hunk 1]
```diff
@@ -6,26 +6,13 @@ export type Library = {
     npmPkgName: string;
 };
 
-/** A snyk.io vulnerability report. */
+/** A snyk.io vulnerability report (just the pieces used by webhint). */
 
 export type Vulnerability = {
-    title: string;
-    moduleName: string;
```
Comment (molant) [inline packages/hint-no-vulnerable-javascript-libraries/src/types.ts]: Can you give an example of `moduleName`?  Maybe we need it for when we analyze `package.json`?

Comment (antross) [reply packages/hint-no-vulnerable-javascript-libraries/src/types.ts]: This appears to be the same as `packageName`. I haven't found an example where it differs yet.

Comment (antross) [reply packages/hint-no-vulnerable-javascript-libraries/src/types.ts]: @molant Also, we can always bring any particular property back if we decide we want to use it. This is about optimizing for what we *currently* use. I added a comment to the jsdoc for `Vulnerability` to make it clear what needs to change if we decide to include anything else in the future.

Any other feedback?

Comment (molant) [reply packages/hint-no-vulnerable-javascript-libraries/src/types.ts]: Not from my side :)

**Pull request merged**